File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/java/cwe-77/code_before3.java
// From cwe-snippets, snippets_2/non-compliant/Java/0181.java

String user = request.getParameter("user");
SMTPSSLTransport transport = new SMTPSSLTransport(session,new URLName(Utilities.getProperty("smtp.server")));
transport.connect(Utilities.getProperty("smtp.server"), username, password);
transport.simpleCommand("VRFY " + user);

File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/python/cwe-77/code_before3.py
# From cwe-snippets, snippets_2/non-compliant/Python/0044.py  

user = request.GET['user']
session = smtplib.SMTP(smtp_server, smtp_tls_port)
session.ehlo()
session.starttls()
session.login(username, password)
session.docmd("VRFY", user)